### PR TITLE
Fix #41377 - Add outdated and upgrade to docs

### LINKED ISSUE #41377: Update USAGE.md with brew cask (outdated|upgrade)
https://github.com/caskroom/homebrew-cask/blob/master/USAGE.md#updatingupgrading-casks

### PR DESCRIPTION
**Pull request by joshghent**
#41377 

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Not really needed as this is a change to the docs - nonetheless all work!

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not already refused in [closed issues].
- [ ] Checked the cask is submitted to [the correct repo].

Again not really needed as this is a documentation change! Let me know if anything more needs doing 👍 

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -73,6 +73,8 @@ Note that `uninstall --force` is currently imperfect. See the man page for more
 * `home` — opens the homepage of the given Cask; or with no arguments, the Homebrew-Cask project page
 * `update` — a synonym for `brew update`
 * `zap` — try to remove *all* files associated with a Cask (may include resources shared with other applications)
+* `outdated` - lists all outdated Casks
+* `upgrade` - updates all outdated Casks; you can view all Casks that will be upgraded by running the `outdated` command
```
Comment (vitorgalvao) [inline USAGE.md]: The section after the `;` can be removed. It’s redundant and explained in the line before.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: USAGE.md (hunk 1 above)
Comment (joshghent) [reply USAGE.md]: @vitorgalvao No problem! Been fixed now 👍

**Comment by vitorgalvao** [comment]
Welcome, and thank you for the contribution.

**Comment by joshghent** [comment]
@vitorgalvao Thanks a lot! Hope to contribute further in the future 👍

**Pull request merged**